src/test/groovy/org/prebid/server/functional/tests/OrtbConverterSpec.groovy

```diff
@@ -1053,7 +1064,55 @@ class OrtbConverterSpec extends BaseSpec {
         then: "BidderRequest should contain the regs.gpp and regs.gppSid as on request"
         verifyAll(bidder.getBidderRequest(bidRequest.id)) {
             regs.gpp == bidRequest.regs.gpp
-            regs.gppSid.eachWithIndex { Integer value, int i -> bidRequest.regs.gppSid[i] == value}
+            regs.gppSid.eachWithIndex { Integer value, int i -> bidRequest.regs.gppSid[i] == value }
+        }
+    }
+
+    def "PBS should remove imp[0].refresh/qty when we don't support ortb 2.6"() {
+        given: "Default bid request with imp[0].refresh/qty"
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            imp[0].tap {
+                refresh = new Refresh(count: PBSUtils.randomNumber, refSettings: [new RefSettings(
+                        refType: PBSUtils.getRandomEnum(RefType),
+                        minInt: PBSUtils.randomNumber)])
+                qty = new Qty(multiplier: PBSUtils.randomDecimal,
+                        sourceType: PBSUtils.getRandomEnum(SourceType),
+                        vendor: PBSUtils.randomString)
```

Comment:
Creating methods to return default objects can improve code reusability. 
However, in case you know that in the future, we will use `refresh` and `qty` only in this tests, it is okay

Reply:
I suppose that currently will enoth use directly init.